qa/rspec/commands/redhat.rb

```diff
@@ -23,11 +23,15 @@ def package_for(filename, base=ServiceTester::Base::LOCATION)
     def install(package, host=nil)
       hosts  = (host.nil? ? servers : Array(host))
       errors = []
+      exit_status = 0
       at(hosts, {in: :serial}) do |_host|
         cmd = sudo_exec!("yum install -y  #{package}")
+        exit_status += cmd.exit_status
         errors << cmd.stderr unless cmd.stderr.empty?
       end
-      raise InstallException.new(errors.join("\n")) unless errors.empty?
+      if exit_status > 0 
```

Comment:
What happened in this exact case is that exit status was 0, but something got written to stderr, so the exception was raised, basically it was a ```Do Nothing``` kind of situation. So for your proposal this would have to raise an exception, what i don't thinks is right. 

However I agree we should add more logging and we can do that certainly in feature changes, works for you?